example/lib/main.dart

```diff
@@ -2,26 +2,36 @@ import 'package:flutter/material.dart';
 import 'package:logging/logging.dart' as log;
 import 'package:matomo_tracker/matomo_tracker.dart';
 
-void main() {
+// See the docker folder for instructions on how to get a
+// test Matomo instance running
+const _matomoEndpoint = 'http://localhost:8765/matomo.php';
+const _sideId = 1;
+const _testUserId = 'Nelson Pandela';
+
+void main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  runApp(MyApp());
+  _setupLogger();
+  await MatomoTracker.instance.initialize(
+    siteId: _sideId,
+    url: _matomoEndpoint,
+    verbosityLevel: Level.all,
+  );
+  MatomoTracker.instance.setVisitorUserId(_testUserId);
+  runApp(const MyApp());
+}
+
+void _setupLogger() {
+  log.Logger.root.level = log.Level.FINEST;
+  log.Logger.root.onRecord.listen((log.LogRecord rec) {
+    debugPrint(
+      '[${rec.time}][${rec.level.name}][${rec.loggerName}] ${rec.message}',
+    );
+  });
```

Comment:
Actually, the whole dependency to the [package logging](https://pub.dev/packages/logging) can be removed (this is some legacy code from the forked project which had a dependency to this package).
You can still add a listener to `MatomoTracker.instance.log` to add informations to the printed outputs.